paperspace/commands/deployments.py

```diff
@@ -1,10 +1,8 @@
-import pydoc
-
 import terminaltables
 
-from paperspace import config, version, client, logger
-from paperspace.commands import CommandBase
-from paperspace.utils import get_terminal_lines
+from paperspace import config, version, client
+from paperspace.commands import common
+from paperspace.commands.common import CommandBase
```

Comment:
why do you keep importing both `common` and `from common`?

Reply:
Missed that, fixed now